.github/workflows/maintenance-desktop-audit.yml

```
@@ -145,30 +148,21 @@ jobs:
-            echo "skipped=true" >> "$GITHUB_OUTPUT"
-            exit 0
-          fi
-          python3 tools/modules/desktops/github/audit_apply.py \
-            --report "${{ github.workspace }}/audit-report.json" \
-            --configng-repo "${{ github.workspace }}/configng" \
-            --summary-output "${{ github.workspace }}/audit-summary.md"
-          echo "skipped=false" >> "$GITHUB_OUTPUT"
+        uses: anthropics/claude-code-action@v1
+        with:
+          claude_code_oauth_token: ${{ secrets.CLAUDE_CODE_OAUTH_TOKEN }}
+          prompt_file: ${{ github.workspace }}/claude-prompt.txt
```

actionlint (1.7.12)

[error] input "prompt_file" is not defined in action "anthropics/claude-code-action@v1". available inputs are "additional_permissions", "allowed_bots", "allowed_non_write_users", "anthropic_api_key", "assignee_trigger", "base_branch", "bot_id", "bot_name", "branch_name_template", "branch_prefix", "classify_inline_comments", "claude_args", "claude_code_oauth_token", "display_report", "exclude_comments_by_actor", "github_token", "include_comments_by_actor", "include_fix_links", "label_trigger", "path_to_bun_executable", "path_to_claude_code_executable", "plugin_marketplaces", "plugins", "prompt", "settings", "show_full_output", "ssh_signing_key", "track_progress", "trigger_phrase", "use_bedrock", "use_commit_signing", "use_foundry", "use_sticky_comment", "use_vertex"

(action)